src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -50,7 +50,7 @@ public static function escapeTrailingBackslash($text)
         if ('\\' === substr($text, -1)) {
             $len = strlen($text);
             $text = rtrim($text, '\\');
-            $text .= str_repeat('<<', $len - strlen($text));
+            $text .= str_repeat('\0', $len - strlen($text));
```

Comment:
Note that currently the PR does not use the null character but a backslash followed by a zero. Wouldn't it be better to actually use the null character (`"\0"` instead of `'\0'`)?